jest.setup.js

```diff
@@ -0,0 +1,6 @@
+// Mock window.URL.createObjectURL for Jest
+if (typeof window.URL.createObjectURL === "undefined") {
+  Object.defineProperty(window.URL, "createObjectURL", {
+    value: jest.fn()
+  });
+}
```

Comment:
Do we need this file?

Reply:
...which leads us to this file, which seems to quell the error I would receive in the logs from attempts to get this whole thing through CI.